release-notes/README.md

```diff
@@ -1,14 +1,21 @@
-# .NET Core Release Notes
```

Comment:
The need for both the README.md and releases.md doesn't seem strong?  Is there a way to eliminate one?  I suppose that is difficult without potential breaking links?

Reply:
I spent a bunch of time thinking about that, when I first added releases.md. I came down to this configuration being best.

https://github.com/dotnet/core/pull/5819#issuecomment-755480375